### ISSUE TITLE
Discrepancy between poe.watch and poe.trade

### ISSUE THREAD

**Issue by russ-** [siegrest/poewatch#51]
I noticed that there's a pretty large discrepancy between the price for L20 +0% quality non-corrupted Winter Orb gems in Synthesis, as reported on poe.watch vs poe.trade, the official PoE trade search, and poe.ninja. The latter three all show that gem is sitting around 12-15c, but poe.watch shows it consistently around 2c. poe.ninja's price history shows it hasn't been below 7c. 

After I noticed this, I checked another random gem -- Herald of Ash -- and see a similar discrepancy. Everything else shows ~9c, but poe.watch shows 4c.

For reference, this is what I see on poe.watch: ![winter-orb-poewatch](https://user-images.githubusercontent.com/622740/55279184-0ae43c00-52ec-11e9-8e59-283d96766743.PNG)

And on poe.trade: ![winter-orb-poetrade](https://user-images.githubusercontent.com/622740/55279185-0ae43c00-52ec-11e9-8e37-2205acbc8932.png)

**Comment by siegrest** [comment]
I see the problem. This probably stems from how gems levels and quality percentages are grouped up.  Eg from the code:
```java
if (level < 19) level = 1;          // lvl       1 = 1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18
else if (level < 21) level = 20;    // lvl      20 = 19,20
// lvl      21 = 21

if (quality < 17) quality = 0;        // quality   0 = 0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16
else if (quality < 22) quality = 20;  // quality  20 = 17,18,19,20,21
else quality = 23;                 // quality  23 = 22,23
```
It would seem that level 19 gems get grouped up with level 20 which drastically lowers their average price. Obviously I didn't put enough thought into this when I first wrote it.

I'll get rid of the ranges so that it'll only include level 20 gems for level 20 range and same for lvl 1.

**Comment by siegrest** [comment]
Fixed in c92ea792f605f5fc30e62bd8810cd9e4265522c6
